src/main/include/NetworkTableUtil.hpp

```diff
@@ -10,38 +10,40 @@
 namespace frc3512::NetworkTableUtil {
 
 /**
- * Creates a NetworkTable entry with a default double value.
+ * Creates a NetworkTable entry with a default double value of 0.
  *
  * @param name         Path of network table entry.
  * @param defaultValue The entry's initial value.
  */
 nt::NetworkTableEntry MakeDoubleEntry(const wpi::Twine& name,
-                                      double defaultValue);
+                                      double defaultValue = 0.0);
 
 /**
- * Creates a NetworkTable entry with a default bool value.
+ * Creates a NetworkTable entry with a default bool value of false.
  *
  * @param name         Path of network table entry.
  * @param defaultValue The entry's initial value.
  */
-nt::NetworkTableEntry MakeBoolEntry(const wpi::Twine& name, bool defaultValue);
+nt::NetworkTableEntry MakeBoolEntry(const wpi::Twine& name,
+                                    bool defaultValue = false);
 
 /**
- * Creates a NetworkTable entry with a default string value.
+ * Creates a NetworkTable entry with a default string value of "none".
  *
  * @param name         Path of network table entry.
  * @param defaultValue The entry's initial value.
  */
 nt::NetworkTableEntry MakeStringEntry(const wpi::Twine& name,
-                                      const wpi::Twine& defaultValue);
+                                      const wpi::Twine& defaultValue = "none");
 
 /**
- * Creates a NetworkTable entry with a default double array value.
+ * Creates a NetworkTable entry with a default double array value of {0, 0, 0}.
  *
  * @param name         Path of network table entry.
  * @param defaultValue The entry's initial value.
  */
 nt::NetworkTableEntry MakeDoubleArrayEntry(
-    const wpi::Twine& name, std::initializer_list<double> defaultValue);
+    const wpi::Twine& name,
+    std::initializer_list<double> defaultValue = {0, 0, 0});
```

Comment:
The default should be an empty array. If certain uses need something different, they can pass it.